From: Louis Royer <55180044+louisroyer@users.noreply.github.com>
Date: Tue, 15 Feb 2022 17:23:08 +0100
Subject: Cleanup CMakeLists.txt

---
 CMakeLists.txt | 14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 08ec1fd..4b576e3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,6 +30,9 @@ target_link_libraries(nr-gnb asn-ngap)
 target_link_libraries(nr-gnb common-lib)
 target_link_libraries(nr-gnb gnb)
 
+install(TARGETS nr-gnb DESTINATION "bin"
+        PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+
 #################### UE EXECUTABLE ####################
 
 add_executable(nr-ue src/ue.cpp)
@@ -39,11 +42,9 @@ target_compile_options(nr-ue PRIVATE -Wall -Wextra -pedantic)
 target_link_libraries(nr-ue common-lib)
 target_link_libraries(nr-ue ue)
 
-###################### IF BINDER ######################
-add_library(devbnd SHARED src/binder.cpp)
-target_compile_options(devbnd PRIVATE -D_GNU_SOURCE -Wall -Wextra)
-target_link_options(devbnd PRIVATE -nostartfiles)
-target_link_libraries(devbnd dl)
+install(TARGETS nr-ue DESTINATION "bin"
+        PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+
 
 #################### CLI EXECUTABLE ####################
 add_executable(nr-cli src/cli.cpp)
@@ -51,3 +52,6 @@ target_link_libraries(nr-cli pthread)
 target_compile_options(nr-cli PRIVATE -Wall -Wextra -pedantic)
 
 target_link_libraries(nr-cli common-lib)
+
+install(TARGETS nr-cli DESTINATION "bin"
+	PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
