From: Louis Royer <55180044+louisroyer@users.noreply.github.com>
Date: Tue, 15 Feb 2022 19:06:36 +0100
Subject: Fix silented format-security warning

---
 src/utils/logger.hpp | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/utils/logger.hpp b/src/utils/logger.hpp
index eab31d4..f5c9777 100644
--- a/src/utils/logger.hpp
+++ b/src/utils/logger.hpp
@@ -41,7 +41,7 @@ class Logger
     template <typename... Args>
     inline void debug(const std::string &fmt, Args &&...args)
     {
-        log(Severity::DEBUG, fmt, args...);
+        log(Severity::DEBUG, fmt.c_str(), args...);
     }
 
     inline void debug(const std::string &fmt)
@@ -52,7 +52,7 @@ class Logger
     template <typename... Args>
     inline void info(const std::string &fmt, Args &&...args)
     {
-        log(Severity::INFO, fmt, args...);
+        log(Severity::INFO, fmt.c_str(), args...);
     }
 
     inline void info(const std::string &fmt)
@@ -63,7 +63,7 @@ class Logger
     template <typename... Args>
     inline void warn(const std::string &fmt, Args &&...args)
     {
-        log(Severity::WARN, fmt, args...);
+        log(Severity::WARN, fmt.c_str(), args...);
     }
 
     inline void warn(const std::string &fmt)
@@ -74,7 +74,7 @@ class Logger
     template <typename... Args>
     inline void err(const std::string &fmt, Args &&...args)
     {
-        log(Severity::ERR, fmt, args...);
+        log(Severity::ERR, fmt.c_str(), args...);
     }
 
     inline void err(const std::string &fmt)
@@ -85,7 +85,7 @@ class Logger
     template <typename... Args>
     inline void fatal(const std::string &fmt, Args &&...args)
     {
-        log(Severity::FATAL, fmt, args...);
+        log(Severity::FATAL, fmt.c_str(), args...);
     }
 
     inline void fatal(const std::string &fmt)
@@ -94,12 +94,12 @@ class Logger
     }
 
     template <typename... Args>
-    inline void log(Severity severity, const std::string &fmt, Args &&...args)
+    inline void log(Severity severity, const char * fmt, Args &&...args)
     {
-        int size = snprintf(nullptr, 0, fmt.c_str(), args...);
+        int size = snprintf(nullptr, 0, fmt, args...);
         std::string res;
         res.resize(size);
-        snprintf(&res[0], size + 1, fmt.c_str(), args...);
+        snprintf(&res[0], size + 1, fmt, args...);
         logImpl(severity, res);
     }
 
