From: Louis Royer <55180044+louisroyer@users.noreply.github.com>
Date: Fri, 25 Feb 2022 22:42:25 +0100
Subject: Use libyaml-cpp-dev

---
 src/utils/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/utils/CMakeLists.txt b/src/utils/CMakeLists.txt
index afaa73f..f4a7f89 100644
--- a/src/utils/CMakeLists.txt
+++ b/src/utils/CMakeLists.txt
@@ -11,3 +11,6 @@ target_link_libraries(utils ext)
 # Debian's libspdlog-dev
 find_package(spdlog CONFIG REQUIRED)
 target_link_libraries(utils spdlog::spdlog_header_only)
+
+# Debian's libyaml-cpp-dev
+target_link_libraries(utils yaml-cpp)
